contrib/gcppubsub/config/203-resolver-clusterrolebinding.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: eventing-sources-gcppubsub-resolver
+subjects:
+- kind: ServiceAccount
+  name: gcppubsub-controller-manager
+  namespace: knative-sources
+roleRef:
```

Comment:
Add a comment that this is defined in knative/eventing (perhaps include a link?).